src/components/Datatable/Pagination/PageButton.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import {
+  getColor,
+  getFontSize,
+  getFontWeight,
+  getLineHeight,
+  pxToRem,
+} from '../../../utils/helpers';
+import { Button } from '../../Button';
+import { ButtonVariants } from '../../Button/Button.enums';
+import { PageButtonProps } from './Pagination.types';
+
+const PageButton = styled(Button)
```

Comment:
Page buttons are missing border-radius. 
Active button probably shouldn't have any hover effect and shouldn't be rendered as `button` tag. 
When I hover on page button it's missing border from Figma

Reply:
active page is now rendered as div and `isActive` prop removed

Reply:
There is no hover state for nav buttons in Figma so I added the border to highlight it because its color as the same as page buttons on hover

Reply:
https://www.figma.com/file/RWVTqwFZk0NepsI3mcZZFD/Platform-Tables?node-id=1058%3A9957

Reply:
Oh you mean Prev/Next buttons, use the same hover effect as `outline` buttons use, it's almost the same styling